backend/tests/esweService.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
rather than ignoring these rules for the whole file it would be better to fix/only apply to offending lines